traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -783,6 +771,32 @@ sub get_print_current_client_connections {
 	( $log_level >> $DEBUG ) && print "DEBUG There are currently $current_connections connections.\n";
 }
 
+sub get_update_status {
+	my $uri     = "/api/1.3/servers/$hostname_short/update_status";
+	my $upd_ref = &lwp_get($uri);
+	if ($upd_ref eq '404') {
+		( $log_level >> $ERROR ) && printf("ERROR ORT version incompatible with current version of Traffic Ops. Please upgrade to Traffic Ops 2.2.\n");
+		exit 1;
+	}
+
+	if ( $upd_ref =~ m/^\d{3}$/ ) {
+		( $log_level >> $ERROR ) && print "ERROR Update URL: $uri returned $upd_ref. Exiting, not sure what else to do.\n";
+		exit 1;
+	}
+
```

Comment:
what if it's a `500`?

Reply:
i 500 will get snagged here, no? so this looks right to me.

Reply:
apparently, if success, $upd_ref will be json. if not success, $upd_ref will be a status code

Reply:
uggh...   sorry -- read that as digits ending in 3 -- not 3 digits..   scratch that..